### ISSUE TITLE
Feature Requests for nVidia Shield Users

### ISSUE THREAD

**Issue by jojodat** [yuliskov/SmartTubeNext#268]
hi, 
for users of NVidia Shield TV, which doesn't have VP9.2 support, it would be good to have an option to ignore all VP9.2 modes, but still choose the highest VP9/AVC video mode according to preset.

And another preset would be good: prefer Multichanel (no matter what format). So if there is a multichanel line ( most likely 5.1), then choose it. If not, then stick to the highest ogg/mp4 or whatever is preferred

**Comment by Jahfry** [comment]
Agreed. Perhaps check if the device in question is a Shield TV and provide a "disable HDR" selection. As long as I don't choose HDR I have no problems using VP9 (Shield Pro 2019) but when I select HDR versions, even very low resolution, I only get audio due to the VP9 implementation. 

I'm not saying I don't WISH HDR worked on YouTube for the Shield ... but ... currently it simply doesn't.
